vllm/sequence.py

```diff
@@ -244,7 +244,7 @@ def get_max_num_running_seqs(self) -> int:
             # candidates running in the future.
             return self.sampling_params.best_of
         else:
-            if self.sampling_params.best_of > self.num_seqs():
+            if self.sampling_params.best_of >= self.num_seqs():
```

Comment:
This fix will make the `if` always be true. Please find the correct change in #1068